joomla/joomla-cms
# [modal] Improve iframe modals: categories, contacts, newsfeeds

Part 2 of iframe inside modals. (see for <code>versions</code> here: https://github.com/joomla/joomla-cms/pull/9995

Note: A one-PR will be done for Hathor overrides (as done in PR for versions #9995)
#### Summary of Changes
- When Bootstrap modal loads an iframe, BS tooltips with placement top are truncated at the top border of the iframe. (no auto placement in BS2). This PR sets the tooltip placement to bottom to fix this issue.
- Remove inline CSS
- Content is embedded in a container div (using already existing container-popup class for modal.php)
- Add empty lines to improve the readability
- <code>newsfeeds</code> add search input description for tooltip (form xml)
#### Testing Instructions (Multilanguages enabled!)
- <code>categories</code> Go to Contents > Categories > open a categorie, and go to Association to select associated category. On click, open the modal changed for categories.
- <code>contacts</code> Go to Components > Contacts > open a contact, and go to Association to select associated contact. On click, open the modal changed for contacts.
- <code>newsfeeds</code> Go to Menus Manager > Add New Menu Item > Type "single newsfeed". Click on select the feed to open the modal changed for newsfeeds.
- For Newsfeeds component, check if in modal (see above) and the list of feeds, the added tooltip description for the search input.

To be testing on latest staging.
